internal/ent/hooks/organization.go

```diff
@@ -395,6 +406,20 @@ func updateUserDefaultOrgOnDelete(ctx context.Context, m *generated.Organization
 	return checkAndUpdateDefaultOrg(ctx, currentUserID, deletedOrgID, m.Client())
 }
 
+// updateOrgSubscriptionOnDelete updates the org subscription to inactive and sets the status to canceled
+func updateOrgSubscriptionOnDelete(ctx context.Context, m *generated.OrganizationMutation) error {
+	deletedOrgID, ok := m.ID()
+	if !ok {
+		return nil
+	}
+
+	if err := m.Client().OrgSubscription.Update().Where(orgsubscription.OwnerID(deletedOrgID)).SetActive(false).SetStripeSubscriptionStatus("canceled").SetExpiresAt(time.Now()).Exec(ctx); err != nil {
```

Comment:
Can you use the stripe constants, particularrly canceled (cancelled) cause that is a common mispelled word - and we are generally using the constants:

```suggestion
	if err := m.Client().OrgSubscription.Update().Where(orgsubscription.OwnerID(deletedOrgID)).SetActive(false).SetStripeSubscriptionStatus(stripe.SubscriptionStatusCanceled).SetExpiresAt(time.Now()).Exec(ctx); err != nil {
```

^ this might not work if you don't have the `stripe` import so don't just take the change :)

Reply:
well, no, because using the type here is inappropriate and it's just a string constant defined by them; plus there's no risk of a typo when the function call has it in there spelled correctly